packages/components/Atoms/ResponsiveImg/ResponsiveImg.vue

```diff
@@ -20,8 +20,7 @@ export default {
   mixins: [breakpoint],
   props: {
     src: {
-      type: String,
-      required: true
```

Comment:
@yeniatencio this doesn't look relevant to the pr, can you explain why this is here?

Reply:
When selecting layout grid view and card display style Thumbnail in BE for the content collection component and if any of the results has no image then it will throw this error. 
<img width="941" alt="Screenshot 2023-06-05 at 6 13 27 pm" src="https://github.com/dpc-sdp/ripple/assets/47239456/1bcb61a9-c11e-45b4-b22a-c106187b18e8">

Reply:
it seems like the src should be required for the image, you've fixed the issue by removing the required prop but the ResponsiveImg component shouldn't be rendered at all. @lambry can I get you to look at this and provide some feedback?

Reply:
Hey @dylankelly @yeniatencio, 

In this instance looks like the card component is being passed an empty object and we aren't checking this is valid, we just assume an object means it's a real image. We might need to update the below check to make sure the object actually has a source, or potentially update the mapping. This sounds like a separate ticket though. 

Cheers

REF: https://github.com/dpc-sdp/ripple/blob/develop/packages/components/Molecules/Card/mixins/card.js#L119